### PR TITLE
Move constant data to a separate single constant folder/file (po-note)

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
Screenshot 2023-02-14 at 2 55 05 AM

### CODE REVIEW

File: src/components/poNotesComponents/poNotesTables/PONotesTable.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    queryParams += query.endDate ? `&endDate=${query.endDate}` : '';
+    return `${DOMAIN}/api/po-notes?${queryParams}`;
+};
+// table for the action items
+export default function PONotesTable(props) {
+    const { heading, definition, accessibilityInformation, query } = props;
+    // need to add page & limit to the query
+    const type = HEADINGS[heading].toUpperCase();
+    const apiUrl = getApiUrl(type, query, 1, 10);
+    console.log("apiUrl >>>", apiUrl);
```
Comment (sugumar-mani) [inline src/components/poNotesComponents/poNotesTables/PONotesTable.js:32]: remove console logs and use implement proper logger utils if needed

File: src/components/poNotesComponents/poNotesTables/PONotesTable.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                        <TableBody >
+                            <TableRow>
+                                {/* Data from get Api call using query params is passed to cardlayout for displaying it in cards */}
+                                <CardLayout chckBox type={HEADINGS[heading]} data={data} /> </TableRow>
+                        </TableBody>
+                    </Table>
+                </TableContainer>
+            </ThemeProvider>
+        </Box>
+
```
Comment (sugumar-mani) [inline src/components/poNotesComponents/poNotesTables/PONotesTable.js:75]: remove extra space

File: src/components/utilityFunctions/Timeline.js [hunk 3]
```diff
@@ -20,7 +20,6 @@ export default function Timeline({ isSubmit, timeline, setTimeline }) {
     </Box>
   );
 }
-
```
Comment (sugumar-mani) [inline src/components/utilityFunctions/Timeline.js:23]: leave an empty line wherever needed. I think we should leave an empty line between the two functions for better readability. Follow what is thought in your code academy

Comment (samim-hossain-mondal) [reply src/components/utilityFunctions/Timeline.js:23]: okay got it.

File: src/components/poNotesComponents/poNotesTables/PONotesTable.js (hunk 1 above)
Comment (bharathraj25) [reply src/components/poNotesComponents/poNotesTables/PONotesTable.js:32]: Yes, removed all console logs

File: src/components/theme/GlobalTheme.js [hunk 4]
```diff
@@ -32,9 +33,10 @@ const theme = createTheme({
         props: { variant: 'h6' },
         style: {
           color: 'white',
-          fontFamily: 'Roboto',
           display: 'flow',
-          lineHeight: '22px'
+          lineHeight: '22px',
+
```
Comment (sugumar-mani) [inline src/components/theme/GlobalTheme.js:38]: unnecessary empty line

**Review by sugumar-mani — APPROVED** [review]
make respective changes in this or next PR and feel free to merge

File: src/components/poNotesComponents/poNotesTables/PONotesTable.js (hunk 2 above)
Comment (bharathraj25) [reply src/components/poNotesComponents/poNotesTables/PONotesTable.js]: removed extra spaces

File: src/components/theme/GlobalTheme.js (hunk 4 above)
Comment (bharathraj25) [reply src/components/theme/GlobalTheme.js]: fixed this

**Pull request merged**